single_tech_samples/databricks/sample2_enterprise_azure_databricks_environment/README.md

```diff
@@ -0,0 +1,273 @@
+
+Below listed are the steps to deploy this sample :
+
+1. Fork and clone this repository. Navigate to (CD) `single_tech_samples/databricks/sample2_enterprise_azure_databricks_environment/`.
+
+1. The sample depends on the following environment variables to be set before the deployment script is run:
+  
+    > - `DEPLOYMENT_PREFIX` - Prefix for the resource names which will be created as a part of this deployment.
+    > - `AZURE_SUBSCRIPTION_ID` - Subscription ID of the Azure subscription where the resources should be deployed.
+    > - `AZURE_RESOURCE_GROUP_NAME` - Name of the containing resource group.
+    > - `AZURE_RESOURCE_GROUP_LOCATION` - Azure region where the resources will be deployed. (e.g. australiaeast, eastus, etc.).
+    > - `DELETE_RESOURCE_GROUP` - Flag to indicate the cleanup step for the resource group.
+
+1. Run '/deploy.sh'
+   > Note: The script will prompt you to log in to the Azure account for authorization to deploy resources.
+
+   The script will validate the ARM templates and the environment variables before deploying the resources. It will also display the status of each stage of the deployment while it executes. The following screenshot displays the log for a successful run:
+
+   > Note: `DEPLOYMENT_PREFIX` for this deployment was set as `lumussample2`
+
+    ![alt text](../Common_Assets/Images/ADE_Script_Deploy.png "Logo Title Text 1")
+
+### 3.3. Deployed Resources
+
+The following resources will be deployed as a part of this sample once the script is executed:
+
+1.Azure Databricks workspace.
+
+![alt text](../Common_Assets/Images/ADE_Adb.png "Logo Title Text 1")
+
+2.Azure Storage with hierarchical namespace enabled.
+
+![alt text](../Common_Assets/Images/ADE_Storage.png "Logo Title Text 1")
+
+3.Azure Key vault with all the secrets configured.
+
+![alt text](../Common_Assets/Images/ADE_Keyvault.png "Logo Title Text 1")
+
+4.Azure Virtual Networks with Vnet peering between the hub and spoke Vnets
+  
+**Hub Vnet**
+![alt text](../Common_Assets/Images/ADE_Hub.png "Logo Title Text 1")
+
+**Spoke Vnet**
+![alt text](../Common_Assets/Images/ADE_Spoke.png "Logo Title Text 1")
+
+5.Azure Firewall with rules configured
+
+**Network Rules**
+![alt text](../Common_Assets/Images/ADE_Firewall.png "Logo Title Text 1")
+
+**Application Rules**
+![alt text](../Common_Assets/Images/ADE_Firewall_AR.png "Logo Title Text 1")
+
+Please note that the firewall rules configured in this sample are as per Microsoft Azure documentation captured [here](https://docs.microsoft.com/en-us/azure/databricks/administration-guide/cloud-configurations/azure/udr). The routes are configured for West US region. In case you need to deploy this sample in any other Azure region, please alter the [ARM template for the firewall](firewall/firewall.template.json) with the IP addresses for the target region.
```

Comment:
I think this is really important step since chances are people will be changing the default region. We need to call this out as an explicit step in the deployment instructions. Something like, "Validate Firewall rules for selected Azure region". Move this up to the deploy steps before running deploy.sh.

Reply:
@namitms Addressing

Reply:
@devlace Resolved